webui/src/App.js

```diff
@@ -617,10 +619,15 @@ function FSRWebUI(props) {
     wsCallbacks.get_cur_profile = function(msg) {
       setActiveProfile(msg.cur_profile);
     };
+    wsCallbacks.thresholds_persisted = function (msg) {
+      setShowPersistedAlert(true);
+      console.log("Persisted thresholds: ", msg.thresholds) // TODO: display them
```

Comment:
Instead of displaying them with the alert, I think it would actually be preferable to draw a line on the bars themselves that indicate the current values saved to the microcontroller. That way users will know whether or not they might need to re-save them as they manipulate the thresholds (since right now I think it's not obvious).